Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorPackages.cs

```diff
@@ -0,0 +1,89 @@
+using UnityEditor;
+using UnityEngine;
+using System;
+using System.Reflection;
+
+namespace PlayFab.PfEditor
+{
+    public class PlayFabEditorPackages : Editor
+    {
+        private const int buttonWidth = 150;
+
+        public static bool IsPubSubInstalled { get { return GetIsPubSubInstalled(); } }
+
+        public static void DrawPackagesMenu()
+        {
+            var labelStyle = new GUIStyle(PlayFabEditorHelper.uiStyle.GetStyle("label"));
+            if(!IsPubSubInstalled)
+            {
+                GUILayout.BeginHorizontal();
+                GUILayout.Label("PersistentSockets: ");
+                if (GUILayout.Button("Install", PlayFabEditorHelper.uiStyle.GetStyle("Button"), GUILayout.MaxWidth(buttonWidth), GUILayout.MinHeight(32)))
+                {
+                    string possibleNewtonsoftPath = "";
+                    if(GetIsNewtonsoftInstalled(out possibleNewtonsoftPath))
+                    {
+                        EditorUtility.DisplayDialog("Newtonsoft is already installed.",
+                            "Please delete your version of Netwonsoft.json.dll in \n\n" + possibleNewtonsoftPath + " \n and retry the install.\n\n compiler conflicts will happen if this stays around", "Continue", "Cancel");
```

Comment:
Compiler conflicts will occur if this package is installed and Newtonsoft already exists.